Libraries/LibWeb/Layout/SVGFormattingContext.cpp

```diff
@@ -176,9 +178,33 @@ void SVGFormattingContext::run(AvailableSpace const& available_space)
     // NOTE: SVG doesn't have a "formatting context" in the spec, but this is the most
     //       obvious way to drive SVG layout in our engine at the moment.
 
+    bool is_standalone_svg_document = !this->context_box()               // the [context] box
+                                           .root()                       // ...of the [root] viewport
+                                           .document()                   // ...of the document
+                                           .page()                       // ...on the page
+                                           .top_level_browsing_context() // ...in the browser
+                                           .active_document()            // the active document (in the browser)
```

Comment:
Question: Would just using the `document()` work? The active document might not be the one that this element is in. (And also raises the issue of it being nullptr.)

Reply:
> Question: Would just using the document() work?

Short answer: No. In case we're laying out a containing document with an svg fragment:
- `...document()` is the document of the svg fragment -- >`svg`
- `...top_level...active_document()` is the containing document -- >`html`

After "-->" above, I show `document_element()->local_name()` for `svg-as-image-implicit-viewbox.html` layout test input.

Reply:
> (And also raises the issue of it being nullptr.)

Good point. Guarding.

Reply:
> The active document might not be the one that this element is in.

Wow! I had not thought of this. The user may load a document (doc1) then immediately switch to a tab with another document (doc2), possibly `about:blank`.  In this case, doc1 may be rendered incorrectly.

Opinion:
- I could not find another way to determine we're in a nested `FormattingContext`.
- `SVGFormattingContext` may be due some love at some point (see my comment above)

Will add a FIXME describing the issue + let you decide whether:
- we should park this PR until streamlining `FormattingContext`
- or you know a way to find whether we're running in a nested context